frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/testProjectAdminPermissions.cy.ts

```diff
@@ -0,0 +1,89 @@
+import yaml from 'js-yaml';
+import {
+  verifyOpenShiftProjectExists,
+  deleteOpenShiftProject,
+  createOpenShiftProject,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/project';
+import { projectDetails, projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+import type { DataScienceProjectData } from '~/__tests__/cypress/cypress/types';
+import { permissions } from '~/__tests__/cypress/cypress/pages/permissions';
+import { ADMIN_USER, TEST_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+
+describe('Verify that users can provide admin project permissions to non-admin users', () => {
+  let testData: DataScienceProjectData;
+  let projectName: string;
+
+  // Setup: Load test data and ensure clean state
+  before(() => {
+    cy.fixture('e2e/dataScienceProjects/dataScienceProject.yaml', 'utf8')
+      .then((yamlContent: string) => {
+        testData = yaml.load(yamlContent) as DataScienceProjectData;
+        projectName = testData.dsOCProjectPermissionsName;
+
+        if (!projectName) {
+          throw new Error('Project name is undefined or empty');
+        }
+
+        return verifyOpenShiftProjectExists(projectName);
+      })
+      .then((exists: boolean) => {
+        if (exists) {
+          cy.log(`Project ${projectName} exists. Deleting before test.`);
+          return deleteOpenShiftProject(projectName).then(() => {
+            cy.log(`Creating project ${projectName} after deletion.`);
+            return createOpenShiftProject(projectName);
+          });
+        }
+        cy.log(`Project ${projectName} does not exist. Creating it now.`);
+        return createOpenShiftProject(projectName);
+      });
```

Comment:
Could we move this to an utils? I think that, probably, we are going to reuse it many times

Reply:
It will be refactored here and on other tests in the future